FirebaseAuth/Tests/SampleSwift/ObjCApiTests/AnonymousAuthTests.m

```diff
@@ -17,7 +17,7 @@
 #import <XCTest/XCTest.h>
 
 #import "FIRAuthApiTestsBase.h"
-#import "FirebaseAuth-Swift.h"
+@import FirebaseAuth;
```

Comment:
curious, was there an issue with the header import?

Reply:
Yes. Do you expect FirebaseAuth-Swift.h to exist in SPM?